<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyWidget_TextMarginsBase</class>
 <widget class="QFrame" name="PropertyWidget_TextMarginsBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>210</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>5</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="4" column="1">
    <widget class="SMScrSpinBox" name="spaceBelow">
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="SMScrSpinBox" name="rightIndent">
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="SMScrSpinBox" name="leftIndent">
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QToolButton" name="resetButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="belowLabel">
     <property name="text">
      <string>Space &amp;Below</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="rightLabel">
     <property name="text">
      <string>&amp;Right Indent</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="firstLabel">
     <property name="text">
      <string>&amp;First Line Indent</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="aboveLabel">
     <property name="text">
      <string>Space &amp;Above</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="SMScrSpinBox" name="spaceAbove">
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="leftLabel">
     <property name="text">
      <string>&amp;Left Indent</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="SMScrSpinBox" name="firstLine">
     <property name="minimum">
      <number>-9999</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SMScrSpinBox</class>
   <extends>QSpinBox</extends>
   <header location="global">ui/smscrspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
